etc/telegraf.conf

```diff
@@ -1337,7 +1337,7 @@
 #   ## Bear in mind that in some serializer a metric even though serialized to multiple
 #   ## lines cannot be split any further so setting this very low might not work
 #   ## as expected.
-#   # max_request_body_size = 1_000_000
+#   # max_request_body_size = 1000000
```

Comment:
You don't need to edit etc/telegraf.conf.  It gets updated once before the release to incorporate changes from all plugins.

Reply:
Ok, noted. Thanks!